geoengine/colorizer.py

```diff
@@ -0,0 +1,107 @@
+"""This module is used to generate geoengine compatible color map definitions as a json string."""
+
+import json
+from typing import Tuple
+import numpy as np
+import matplotlib.pyplot as plt
+from matplotlib.colors import ListedColormap
+from matplotlib.cm import ScalarMappable
+
+
+class Colorizer():
+    """This class is used to generate geoengine compatible color map definitions as a json string."""
+
+    # pylint: disable=too-few-public-methods
```

Comment:
Is this necessary? You have a public method.

Reply:
Pylint wants 2 public methods.
Error: 

> ************* Module geoengine.colorizer
geoengine/colorizer.py:11:0: R0903: Too few public methods (1/2) (too-few-public-methods)